src/validator.cpp

```diff
@@ -2612,7 +2608,7 @@ IdMap Validator::ValidatorImpl::buildModelIdMap(const ModelPtr &model)
                 addIdMapItem(id, info, idMap);
             }
         }
-        if (units->isImport() && units->importSource() != nullptr && !units->importSource()->id().empty()) {
+        if (units->importSource() != nullptr && !units->importSource()->id().empty()) {
```

Comment:
Forgot that one:
```c++
if ((units->importSource() != nullptr) && !units->importSource()->id().empty()) {
```

Reply:
[9a5d2f1](https://github.com/cellml/libcellml/pull/1128/commits/9a5d2f1c22b12b128dc26d904a200122b1d3a678)